connectors/netty-connector/src/main/java/org/glassfish/jersey/netty/connector/NettyClientProperties.java

```diff
@@ -53,4 +53,6 @@
      *  </p>
      */
     public static final String MAX_CONNECTIONS = "jersey.config.client.maxConnections";
+
+    public static final String ENABLE_SSL_HOSTNAME_VERIFICATION = "jersey.config.client.tls.enableHostnameVerification";
```

Comment:
Not sure if there is a common client config setting. This mimics the setting as in Jetty connector.